src/helpers/cauldron/position/getUserBorrowInfo.ts

```diff
@@ -0,0 +1,87 @@
+import { BigNumber } from "ethers";
```

Comment:
In theory, this can all be replaced by the MarketLens, right? Should be inside the userInfo call.